README.md

```diff
@@ -57,17 +57,17 @@ module.exports = function(el) {
 };
 ```
 
-Now whenever you add an element with `data-wt="popup"` to the DOM, a click listener will be bound:
+Now whenever you add an element with `data-wt="alert"` to the DOM, a click listener will be bound:
 
 ```
-var content = $.get(someUrl); // Contains a "popup" block
-$('body').append(content); // "popup" block is initialized automatically
+var content = $.get(someUrl); // Contains an "alert" block
+$('body').append(content); // "alert" block is initialized automatically
 ```
 
 You can access the return value of the initialization function through your `Regulator` instance:
 
 ```
-myRegulator.withController($("[data-wt='AlertButton']"), function(controller) {
```

Comment:
it's possible that `AlertButton` was correct here. That style of casing maybe implies an `AlertButton` object/prototype/etc defined somewhere? Doesn't appear that way, though, as that's an HTML attribute search.